vespalib/src/tests/stllike/hashtable_test.cpp

```diff
@@ -155,6 +155,32 @@ TEST("benchmark clear") {
     }
 }
 
+class NonPOD {
+public:
+    NonPOD() : _v(rand()) {}
+    ~NonPOD() { _v = 0; }
+private:
+    uint32_t _v;
+};
+
+/**
+ * Performance is identical for NonPOD objects as with POD object.
+ * Object are are only constructed on insert, and destructed on erase/clear.
+ */
+TEST("benchmark hash table reconstruction with POD objects") {
```

Comment:
This should probably be "with _non_-POD objects"

Reply:
Fixed